src/plugins/data_table/column_config_generator.js

```diff
@@ -31,6 +31,11 @@ var ColumnConfigGenerator =  Backdraft.Utils.Class.extend({
       }
     });
 
+    // make the bulk column the first one if present
+    this.columnsConfig = _.sortBy(this.columnsConfig, function (columnConfig) {
```

Comment:
looks like we need to tweak HoundCI to know about some global variables. @HParker can you point me to where we can do that?